targets/lenovo-x1-carbon.nix

```diff
@@ -449,6 +462,7 @@
                     extraModules = [
                       {
                         time.timeZone = "Asia/Dubai";
+                        security.pki.certificateFiles = [../modules/virtualization/microvm/mitmproxy-ca/mitmproxy-ca-cert.pem];
```

Comment:
should be excluded if disabled

Reply:
Fixed in [b500944](https://github.com/tiiuae/ghaf/commit/b500944e5baef64e8efefa4b2e5fbbedcdbbbce5)